### PR TITLE
Pull Request for Issue 2443: Adjustments to PGM XAS scan configuration view.

### PR DESCRIPTION
**Pull request by mButlerDM**
Added more spacing to the sides of the view.

Adjusted the name filed so it was nested into the left layout.

Added a label specifying the name field was for the name.

Added an estimation per scan field underneath the name field as it is on other beamlines.

#2443

### CODE REVIEW

**Comment by mButlerDM** [comment]
@XuetaoLiu-CLS Could you review when you have a chance?

**Comment by XuetaoLiu-CLS** [comment]
Looks good. Time Estimate: 5 minutes

File: source/ui/PGM/PGMXASScanConfigurationView.cpp [hunk 1]
```diff
@@ -11,14 +11,37 @@ PGMXASScanConfigurationView::PGMXASScanConfigurationView(PGMXASScanConfiguration
 {
 	configuration_ = configuration;
 
+	// Scan Name
+
+	QLabel *scanNameLable = new QLabel("Scan Name: ");
```
Comment (dretrex) [inline source/ui/PGM/PGMXASScanConfigurationView.cpp]: scanName**_Label_**

**Comment by dretrex** [comment]
If you change the number of iterations, does the time in the configuration view holder update accordingly?

**Comment by dretrex** [comment]
Only a couple comments/questions.  Looks good otherwise.

Time Estimate: 3 minutes

File: source/ui/PGM/PGMXASScanConfigurationView.cpp (hunk 1 above)
Comment (XuetaoLiu-CLS) [inline source/ui/PGM/PGMXASScanConfigurationView.cpp]: :bomb: good catch

**Comment by XuetaoLiu-CLS** [comment]
@dretrex I think the estimate here is only about a single scan. We didn't do the estimation for the iterated scans in the other beamlines as well.

**Comment by dretrex** [comment]
Since the scan configuration view holder displays the estimated time, we should connect it in this pull request as well.  At the moment, you should only need to emit a signal from AMScanConfiguration and it should do the rest.

**Comment by mButlerDM** [comment]
Seems reasonable I will look into making that change.

**Comment by mButlerDM** [comment]
@dretrex Got that change in, let me know if that is what you had in mind.

**Comment by dretrex** [comment]
Looks good!

**Pull request merged**